### DISCUSSION TITLE
Error when opening /cp

### DISCUSSION THREAD

**Discussion by aalainn** [statamic/cms#4597] [Q&A] [answered]
Hi
I run the local server with "php artisan serve".
Then I open the site in the Browser with the URL http://127.0.0.1:8000

Envoking http://127.0.0.1:8000/cp throws the following error:
cURL error 60: SSL certificate problem: unable to get local issuer certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://outpost.statamic.com/v3/query

What do I do wrong?

**Comment by jackmcdade** [comment, marked as answer]
`artisan serve` is a very "underpowered" way to serve a site. If you have the option, I highly recommend using another method. Maybe [Laragon](https://laragon.org/download)?

**Reply by aalainn** [reply]
Ok, I switched to Laragon an it works :-)

The files for a Statamic project are somewhere on my local disk.
I the documentation of Statamic is written, that you can envoke a project by entering "project-name.test" in the Browser.
But it does not work in my case.
I have to run "php artisan serve" to envoke it via http://127.0.0.1:8000

What do I do wrong there?

**Comment by jasonvarga** [comment]
It's an issue with SSL and curl on Windows.
Check out this answer on Stack Overflow. https://stackoverflow.com/a/21114601

**Reply by aalainn** [reply]
I have seen that, but I can't find any cacert.pem-file in the wamp-directory.

**Reply by jasonvarga** [reply]
Click the link in the stack overflow answer to download it, and put it somewhere.
